### PR TITLE
Disk image refs

### PR DESCRIPTION
**Pull request by adampbeardsley**
This should be merged after astrochart/CHART#233 (ahem, @dannyjacobs ).

### CODE REVIEW

**Review by lmberkhout — APPROVED** [review]

File: wiki/telescope_design.md [hunk 1]
```diff
@@ -114,7 +114,9 @@ can also use a laptop. We use a pi because it is cheaper and easier to maintain.
 We provide a ready to run SD card image which can get you started.
 
 1. Download the latest [CHART Pi image](https://galileo.sese.asu.edu/chart/).
+  - If you're ambitious and want to install the software from scratch, you can follow our "Creating a Disk Image" instructions in the [github repo](https://github.com/astrochart/CHART).
```
Comment (dannyjacobs) [inline wiki/telescope_design.md:117]: I would give more guidance than "if you're ambitious" how about "those wishing to diverge from the Pi setup will need to install the libraries. This is the path for hackers who want to help us fix bugs, make new features, or install on some other computer besides a Raspberry Pi. "

File: wiki/telescope_design.md [hunk 2]
```diff
@@ -125,9 +127,17 @@ To get started, we need to make sure that all our wires are hooked up and ready
 
 ![pi setup](assets/rasp_set_up.jpeg){:height="50%" width="50%"}
 
+The CHART Pi image comes with all the software you need to run CHART and analyze data.
```
Comment (dannyjacobs) [inline wiki/telescope_design.md:130]: Lets be more specific than "run CHART".  How about "The CHART Pi image comes pre setup ready to observe the 21cm line and analyze the data."

File: wiki/telescope_design.md [hunk 3]
```diff
@@ -125,9 +127,17 @@ To get started, we need to make sure that all our wires are hooked up and ready
...
+```bash
+cd ~/CHART
+git pull
+pip install .
+```
+
 ### Test your setup
 
-Click the `chart-observe` link on the Desktop of the pi, then click "execute in terminal." Success will look like a window opening with some boxes asking for input. 
+Click the `chart-observe` link on the Desktop of the pi, then click "execute in terminal." Success will look like a window opening with some boxes asking for input.
```
Comment (dannyjacobs) [inline wiki/telescope_design.md:140]: Is this a rightclick?

**Review by dannyjacobs — COMMENTED** [review]
suggestions to help reduce questions

**Pull request merged**